requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest~=6.2.4
-mock~=4.0.3
 argparse~=1.4.0
 retry~=0.9.2
+setuptools~=57.0.0
```

Comment:
Is there a particular reason for using ~= here? I'd say we don't need to specify dependency versions at all.